cmd/kogito/command/converter/infinispanproperties_converter.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
Please rename to `infinispanmeta_converter.go` so we have coherence with `kafkameta` ?

Reply:
done